test/core/event_engine/event_engine_test_utils.h

```diff
@@ -132,7 +131,38 @@ class ConnectionManager {
   std::unique_ptr<EventEngine> oracle_event_engine_;
 };
 
+void AppendStringToSliceBuffer(SliceBuffer* buf, absl::string_view data);
+
+std::string ExtractSliceBufferIntoString(SliceBuffer* buf);
```

Comment:
I think ExtractSliceBufferIntoString is declared twice: here and in line 136.

Reply:
Good catch, fixed